tournament/on.cfg

```diff
@@ -33,6 +33,18 @@ sv_only_hook_kills "0"
 sv_damage_needed_for_kill "4"
 sv_fastcap "0"
 
+add_map_to_pool "ctf2"
+add_map_to_pool "ctf3"
+add_map_to_pool "ctf5_limited"
+add_map_to_pool "ctf4_old"
+add_map_to_pool "ctf_ambiance_limited"
+add_map_to_pool "ctf_duskwood_limited"
+add_map_to_pool "ctf_tantum"
+add_map_to_pool "ctf_mine"
+add_map_to_pool "ctf_planet_limited"
+add_map_to_pool "ctf_moon_limited"
+add_map_to_pool "ctf_mars"
```

Comment:
I am not sure if this belongs here. This either needs a clear map pool on top or should be put in a place that is only loaded once. Otherwise you get all entries doubled every time the tournament gets turned on.

Also the feature is not on the main branch yet.

Reply:
Okay update. I just merged the map pool feature including the ``clear_map_pool`` command.
I will add it my self on the top then it can stay in this file. Nothing you have to do.